src/main/java/net/dries007/tfc/objects/blocks/devices/BlockFirePit.java

```diff
@@ -90,6 +89,20 @@ public boolean isFullCube(IBlockState state)
         return false;
     }
 
+    /**
+     * Determines if this block should set fire and deal fire damage
+     * to entities coming into contact with it. See #1327
+     *
+     * @param world The current world
+     * @param pos   Block position in world
+     * @return True if the block should deal damage
+     */
+    @Override
+    public boolean isBurning(IBlockAccess world, BlockPos pos)
+    {
+        return world.getBlockState(pos).getActualState(world, pos).getValue(LIT);
```

Comment:
Don't use `getActualState`

Reply:
This is consistent with the usage in BlockPitKiln. Please clarify whether BlockPitKiln should be updated to not do so. What's the drawback?

Reply:
- Actual states in 1.12 are a "only use if necessary to escape the 16-variant block metadata limit" as they are removed in future versions.
 - Pit kilns used to have actual states (which is why it queries it). They were removed (760d09e5802ff2cb8c0847fb7a525b1d68ddd464) and so should not be using it any more.
 - Fire pits have never had actual states and so should not be querying it.